tests/all/js_globals/Object.rs

```diff
@@ -54,3 +54,35 @@ fn has_own_property() {
         "#)
         .test()
 }
+
+#[test]
+fn to_string() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js;
+
+            #[wasm_bindgen]
+            pub fn to_string(obj: &js::Object) -> String {
+                obj.to_string()
+            }
+
+            #[wasm_bindgen]
+            pub fn test() {
```

Comment:
Oops, should have caught this earlier. The `test` function in a `.rs` is only called by default if you don't supply a `test.ts` file. So in this case, this rust `test` is never called. This should be added to the `test.ts` file:

```js
wasm.test();
```

Reply:
Good call! The error message that you get from wasm, is a lot less user friendly. The sasert statement in rust should now run and pass.

Reply:
Yes, this is what https://github.com/rustwasm/console_error_panic_hook is for, but we haven't hooked it up into the tests. I'll file an issue!